process/block/metablock.go

```diff
@@ -261,18 +261,27 @@ func (mp *metaProcessor) checkAndRequestIfShardHeadersMissing(round uint64) {
 	return
 }
 
-func (mp *metaProcessor) indexBlock(metaBlock *block.MetaBlock, headerPool map[string]*block.Header) {
+func (mp *metaProcessor) indexBlock(
+	metaBlock data.HeaderHandler,
+	headerPool map[string]*block.Header,
+	lastMetaBlock data.HeaderHandler,
+) {
 	if mp.core == nil || mp.core.Indexer() == nil {
 		return
 	}
-
 	// Update tps benchmarks in the DB
 	tpsBenchmark := mp.core.TPSBenchmark()
 	if tpsBenchmark != nil {
 		go mp.core.Indexer().UpdateTPS(tpsBenchmark)
 	}
 
-	//TODO: maybe index metablocks also?
+	publicKeys, err := mp.nodesCoordinator.GetValidatorsPublicKeys(metaBlock.GetPrevRandSeed(), metaBlock.GetRound(), sharding.MetachainShardId)
+	if err != nil {
+		return
+	}
+	signersIndexes := mp.nodesCoordinator.GetValidatorsIndexes(publicKeys)
+
+	saveRoundInfoInElastic(mp.core.Indexer(), mp.nodesCoordinator, sharding.MetachainShardId, metaBlock, lastMetaBlock, signersIndexes)
```

Comment:
add empty line before `signerIndexes` and remove empty line before `saveRoundInfo..`

Reply:
done.

Reply:
not done yet

Reply:
done